FirstProject/css/main.css

```diff
@@ -0,0 +1,154 @@
+
+/*Header section*/
+header {
+    background-image: url(../img/feedback-banner.jpg);
+    width: 100%;
+}
+
+/*Body section*/
+body {
+    overflow-x:hidden;
+}
+
+.left_image {
```

Comment:
Ok. I will modify and in the future I will use '-" to name the classes.

thanks 👍